pom.xml

```diff
@@ -3,7 +3,7 @@
     <groupId>io.github.kobylynskyi</groupId>
     <artifactId>graphql-codegen-maven-plugin</artifactId>
     <packaging>maven-plugin</packaging>
-    <version>1.2.4-SNAPSHOT</version>
+    <version>1.3.0</version>
```

Comment:
Please update the version to 1.3.0-SNAPSHOT (maven-publish plugin requires that)

Reply:
Done